src/sentry/lang/native/applecrashreport.py

```diff
@@ -17,6 +18,12 @@
 REPORT_VERSION = "104"
 
 
+# symbolic.common.parse_addr with instrumentation
+@sentry_sdk.trace
+def parse_addr(x: int | str | None) -> int:
+    return symbolic.common.parse_addr(x)
```

Comment:
The amount of times this function is called, this will produce a very large amount of spans, It may make more sense to track this differently.